php/Dockerfile-7.0

```diff
@@ -24,6 +24,8 @@ RUN apt-get update -q -y \
         libicu52 \
         libxslt1.1 \
         libmemcachedutil2 \
+# Required by eZ Commerce
```

Comment:
According to [these requirements](https://github.com/ezsystems/ezplatform/blob/2.3/composer.json#L27) eZ Commerce requires php 7.1

Reply:
It does, but as said on other PR I wanted (supported) images to be in sync